### PR TITLE
Document annotation service.beta.kubernetes.io/aws-load-balancer-manage-backend-security-group-rules

### PR DESCRIPTION
**Pull request by niranjandarshann**
Annotated service.beta.kubernetes.io/aws-load-balancer-manage-backend-security-group-rules not documented.

### CODE REVIEW

**Comment by niranjandarshann** [comment]
This PR Fixes issue #42880

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -1726,6 +1726,9 @@ Example: `service.beta.kubernetes.io/aws-load-balancer-manage-backend-security-g
 
 Used on: Service
 
+The cloud controller manager integration with AWS elastic load balancing configures
+a load balancer based on this annotation. It determines if the controller should enable automatic setup of the instance/ENI security group to permit access to the ports used for both  traffic and healthcheck.
```
Comment (windsonsea) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1730]:
```suggestion
a load balancer based on this annotation. It determines if the controller should enable
automatic setup of the instance/ENI security group to permit access to the ports used
for both  traffic and healthcheck.
```
a nit about wrapping for consistency in context

**Comment by windsonsea** [comment]
Please squash your commits into a single one: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by windsonsea** [comment]
Sure, need a technical review.
/lgtm

**Comment by sftim** [comment]
/retitle Document annotation service.beta.kubernetes.io/aws-load-balancer-manage-backend-security-group-rules

**Comment by sftim** [comment]
/lgtm cancel

`service.beta.kubernetes.io/aws-load-balancer-manage-backend-security-group-rules` already seems to be documented

**Comment by niranjandarshann** [comment]
> /lgtm cancel
> 
> `service.beta.kubernetes.io/aws-load-balancer-manage-backend-security-group-rules` already seems to be documented

@sftim  It is there but the `use on` description of the annotation is not present as in severals. I think description of the annotation will be helpful  for user and looks good as other annotations have. Only  Redirecting to the link for use of annotation is some how not looks good.
Hoping for the suggestion and your view still if not required then i will close the PR marking as not planned.

**Comment by sftim** [comment]
I'm happy with the description in https://kubernetes.io/docs/reference/labels-annotations-taints/#service-beta-kubernetes-io-aws-load-balancer-manage-backend-security-group-rules and the link to the AWS load balancer controller docs.

**Comment by niranjandarshann** [comment]
> I'm happy with the description in https://kubernetes.io/docs/reference/labels-annotations-taints/#service-beta-kubernetes-io-aws-load-balancer-manage-backend-security-group-rules and the link to the AWS load balancer controller docs.

closing this PR as the issue was not planned.
/close

**Pull request closed without merging**